automatic/k9s/update.ps1

```diff
@@ -34,7 +34,11 @@ function global:au_GetLatest {
 
   $checksumAsset = $domain + ($download_page.Links | ? href -match "checksums\.txt$" | select -first 1 -expand href)
   $checksum_page = Invoke-WebRequest -Uri $checksumAsset -UseBasicParsing
-  $filename64 = -join('k9s_v', $version, '_Windows_x86_64.tar.gz')
+  if($VersionStyle -eq 0) {
```

Comment:
Not sure If I like that a global `VersionStyle` variable is used for this.

Is it possible to first parsing the checksum with the first style, then if that fails/is empty then retry the same checksum regex with the different style?

Reply:
I am not sure if I understand this. The reason I had to use a global variable was the fact that I couldn't pass a value or context into the `au_GetLatest` method. Also the method "return" the url string which "supposedly" is handled by AU somehow. Hence I was passing a context from the global scope, which is triggering the update call, into the method using a global variable.

Reply:
@SayakMukhopadhyay
Yes, I meant to post it here, sorry for the confusion.
AU requires a version to be passed in order to determine if the package is updated.
You are using the VersionStyle variable to determine the filename. Using the helper function `Get-Version` to get the version from the URL would eliminate the step of using your VersionStyle variable in the global scope. This would in turn make @admiringworm happier about nixing the global variable.